device/cluster.cpp

```diff
@@ -172,10 +172,9 @@ bool Cluster::is_tlb_mapped(tt_cxy_pair target, uint64_t address, uint32_t size_
     auto* dev = get_tt_device(target.chip);
 
     int32_t tlb_index = map_core_to_tlb_per_chip.at(target.chip).at(tt_xy_pair(target.x, target.y));
-    auto tlb_description = dev->get_architecture_implementation()->describe_tlb(tlb_index);
+    auto tlb_description = dev->get_architecture_implementation()->get_tlb_configuration(tlb_index);
```

Comment:
I think replacing auto with exact type here (tlb_configuration) would be more readable